lti_consumer/tests/unit/test_lti_xblock.py

```diff
@@ -132,13 +142,43 @@ def test_context_id(self):
         """
         self.assertEqual(self.xblock.context_id, str(self.xblock.course_id))
 
-    def test_validate(self):
+    @ddt.data(*VALID_CUSTOM_PARAMETERS)
+    def test_validate_with_valid_custom_parameters(self, custom_parameters):
         """
-        Test that if custom_parameters is empty string, a validation error is added
+        Test if all custom_parameters item are valid
         """
-        self.xblock.custom_parameters = ''
+        self.xblock.custom_parameters = custom_parameters
         validation = self.xblock.validate()
-        self.assertFalse(validation.empty)
+        self.assertEqual(validation.messages, [])
+
+    @patch('lti_consumer.lti_xblock.ValidationMessage')
+    @patch.object(Validation, 'add')
+    def test_validate_with_empty_custom_parameters(self, add_mock, mock_validation_message):
+        """
+        Test if custom_parameters is not a list, a validation error is added
+        """
+        mock_validation_message.ERROR.return_value = 'error'
+        self.xblock.custom_parameters = ''
+        self.xblock.validate()
```

Comment:
missing blank line. @kuipumu 
```suggestion
        self.xblock.custom_parameters = ''
        
        self.xblock.validate()
```